crates/topos-certificate-spammer/src/error.rs

```diff
@@ -16,4 +16,6 @@ pub enum Error {
     TCENodeConnection(topos_tce_proxy::Error),
     #[error("Certificate signing error: {0}")]
     CertificateSigning(topos_core::uci::Error),
+    #[error("Kubernetes config error: {0}")]
```

Comment:
I feel that it can be misleading, we're not dealing with `kubernetes` config at all here.
Maybe we should rename the `--kubernetes` into `--benchmark` or something similar?